sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -234,7 +234,7 @@ public BlobServiceClient GetServiceClient_PremiumBlobAccount_SharedKey()
             }
 
             BlobContainerClient container = InstrumentClient(service.GetBlobContainerClient(containerName));
-            await container.CreateAsync(metadata: metadata, publicAccessType: publicAccessType);
+            await container.CreateIfNotExistsAsync(metadata: metadata, publicAccessType: publicAccessType);
```

Comment:
There are no other direct `Create` calls?  Interesting...  I guess it's only blobs we do that for.

Reply:
There are for places where we are actually testing the Create API and such. There may be some other one off uses as well, but I figured I would start with this change.

Reply:
I think we should first check if the container exists, and throw if it does.  This will cause failures if we generate a duplicate GUID (which we definitely want to catch).

Reply:
I wouldn't want to slow down every single test for this. I wonder if we can investigate this separately - whether GUID collisions are still possible with our test framework.

Reply:
I will create a separate PR to see how much this changes overall test runtime.